rental-angular/src/app/components/home/home.component.html

```diff
@@ -10,38 +10,38 @@ <h4>Dane firmy</h4>
       </thead>
       <tbody>
       <tr>
-        <td class="tdWidth20">     <label >Nazwa:</label></td>
+        <td class="tdWidth20">     <label >Nazwa: </label></td>
         <td><p>{{companyData?.name}}</p>
         </td>
       </tr>
       <tr>
         <td>
-          <label>Domena:</label>
+          <label>Domena: </label>
 
         <td>   <p><a href="{{companyData?.webDomain}}">{{companyData?.webDomain}}</a></p>     <td>
       </td>
       </tr>
       <tr>
-        <td>      <label >Adres:</label>
+        <td>      <label >Adres: </label>
         </td>
         <td><p>ul. {{companyData.address?.streetWithNumber}}, {{companyData.address?.postalCode}} {{companyData.address.city}} </p></td>
```

Comment:
to powinno wyglądać tak: companyData?.address?.streetWithNumber. Znak zapytania sprawdza czy nie ma nulla w obiekcie przed nim